### PR TITLE
Feature/issues list

### PR DESCRIPTION
**Pull request by forever-Agriculture**
## A simple list of issues
*Features:*
* the list's based on a title, category and description columns;
* got empty links for now (will be added in the next task of showing a detailed issue)
* got a simple pagination
* got a simple sorting integrated into the pagination as well

*Warning, do not mock me because of the design, i tried being creative:)*

### CODE REVIEW

**Review by alevitsky — APPROVED** [review]

File: client/basic/urls.py [hunk 1]
```diff
@@ -20,11 +20,13 @@
 from django.views.generic import RedirectView
 from city_issues.views import (
     edit_issue_view, get_all_issues_data, get_issue_data,
-    HomePageView, map_page_view, IssueCreate)
+    HomePageView, map_page_view, IssueCreate, CheckIssues)
```
Comment (KostSvitlana) [inline client/basic/urls.py]: Imports is not alphabetical

File: client/apps/city_issues/templates/issues_list.html [hunk 2]
```diff
@@ -0,0 +1,73 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load static from staticfiles %}
+
+{% block meta_title %}All Issues:{% endblock meta_title %}
```
Comment (KostSvitlana) [inline client/apps/city_issues/templates/issues_list.html]: I think here is no need colon

**Review by KostSvitlana — COMMENTED** [review]
Maybe add column status

File: client/apps/city_issues/templates/issues_list.html (hunk 2 above)
Comment (forever-Agriculture) [reply client/apps/city_issues/templates/issues_list.html]: fixed

File: client/basic/urls.py (hunk 1 above)
Comment (forever-Agriculture) [reply client/basic/urls.py]: fixed

**Review by KostSvitlana — APPROVED** [review]

**Pull request merged**